src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/commands.py

```diff
@@ -249,7 +54,292 @@ def load_command_table(self, _):
+
+        g.storage_command('list', 'list_blobs', transform=transform_storage_list_output,
+                          table_transformer=transform_blob_output)
+        g.storage_command('download', 'get_blob_to_path')
+        g.storage_command('generate-sas', 'generate_blob_shared_access_signature')
+        g.storage_command('url', 'make_blob_url', transform=transform_url)
+        g.storage_command('snapshot', 'snapshot_blob')
+        g.storage_command('show', 'get_blob_properties', table_transformer=transform_blob_output,
+                          exception_handler=g.get_handler_suppress_404())
+        g.storage_command('update', 'set_blob_properties')
+        g.storage_command('exists', 'exists', transform=create_boolean_result_output_transformer('exists'))
+        g.storage_command('delete', 'delete_blob', transform=create_boolean_result_output_transformer('deleted'),
+                          table_transformer=transform_boolean_for_table)
+
+        g.storage_custom_command('set-tier', 'set_blob_tier')
+        g.storage_custom_command('upload', 'upload_blob',
+                                 doc_string_source='blob#BlockBlobService.create_blob_from_path')
+        g.storage_custom_command('upload-batch', 'storage_blob_upload_batch')
+        g.storage_custom_command('download-batch', 'storage_blob_download_batch')
+        g.storage_custom_command('delete-batch', 'storage_blob_delete_batch')
+
+        g.storage_command('metadata show', 'get_blob_metadata', exception_handler=g.get_handler_suppress_404())
+        g.storage_command('metadata update', 'set_blob_metadata')
+
+        g.storage_command('lease acquire', 'acquire_blob_lease')
+        g.storage_command('lease renew', 'renew_blob_lease')
+        g.storage_command('lease release', 'release_blob_lease')
+        g.storage_command('lease change', 'change_blob_lease')
+        g.storage_command('lease break', 'break_blob_lease')
+
+        g.storage_command('copy start', 'copy_blob')
+        g.storage_command('copy cancel', 'abort_copy_blob')
+        g.storage_custom_command('copy start-batch', 'storage_blob_copy_batch')
+
+    with self.command_group('storage blob incremental-copy',
+                            command_type=CliCommandType(
+                                operations_tmpl='azure.multiapi.storage.blob.pageblobservice#PageBlobService.{}',
+                                client_factory=page_blob_service_factory,
+                                resource_type=ResourceType.DATA_STORAGE)) as g:
+        g.storage_command('start', 'incremental_copy_blob')
+
+    with self.command_group('storage blob incremental-copy',
+                            command_type=CliCommandType(
+                                operations_tmpl='azure.multiapi.storage.blob.blockblobservice#BlockBlobService.{}',
+                                client_factory=page_blob_service_factory,
+                                resource_type=ResourceType.DATA_STORAGE)) as g:
```

Comment:
Why are you defining a `CliCommandType` inline?  The whole point of it is to be a reusable bundle of kwargs and used in this way, it can't be reused.  You could simply list out the kwargs.

Reply:
Please these two commands are special and this SDK is only used for this command group.

Reply:
In that case you should just be able to put the kwargs.
```Python
with self.command_group('blah', operations_tmpl=..., client_factory=..., resource_type=...)
```

We used to have this a lot with CliArgumentType where people would define them inline and it didn't make sense.  If puttings the raw kwargs here doesn't work, please let me know.

Reply:
ok